src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid $color-gray-elements;
+  border-radius: 5px;
+
+  @include smooth-hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__product-name {
+    margin-bottom: 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    transition: 0.3s;
```

Comment:
Specify the exact properties you want to apply transition to. Also, would be great to create a variable for transition-duration